examples/pytorch/gcn/gcn_mp.py

```diff
@@ -49,73 +37,79 @@ def forward(self, nodes):
             h = self.activation(h)
         return {'h': h}
 
-
 class GCNLayer(nn.Module):
```

Comment:
Similarly, let's remove this module entirely. Just use DGL's built-in `GraphConv` module.

Reply:
Removed this implementation entirely.